addons/sms/models/mail_thread.py

```diff
@@ -264,9 +266,12 @@ def _notify_record_by_sms(self, message, recipients_data, msg_vals=False,
         sms_all = self.env['sms.sms'].sudo()
 
         # pre-compute SMS data
-        body = msg_vals['body'] if msg_vals and msg_vals.get('body') else message.body
+        body = sms_content or html2plaintext(html_sanitize(
+            msg_vals['body'] if msg_vals and msg_vals.get('body') else message.body,
+            sanitize_extra_tags=['a'])  # avoid creating footnotes
```

Comment:
without that option, `test_sms_composer.py` is green, is it really useful ?
```suggestion
            )  # avoid creating footnotes

Reply:
My bad, the tests were incomplete.
Then, I stumbled uppon the case of an html content passed with a link inside, and in that case, we can't just remove it or the sms would include the label instead of the url...
Please have a look at this update.